src/sentry/static/sentry/app/views/eventsV2/utils.tsx

```diff
@@ -303,6 +303,16 @@ export function downloadAsCsv(tableData, columnOrder, filename) {
     data: data.map(row =>
       headings.map(col => {
         col = getAggregateAlias(col);
+        // This needs to match the order done in the userBadge component
```

Comment:
This was made to match the userBadge component, but I couldn't figure out a way to yank out the code there into a function since the objects they take look pretty different.